# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/units/gcc33-support.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- units-1.80/units.c~	2002-05-07 00:20:38.000000000 +0200
+++ units-1.80/units.c	2003-08-12 23:47:18.000000000 +0200
@@ -2358,7 +2358,7 @@
 
 #if 0
 
-    "sed 's/.*/& ft\"
+    "sed 's/.*/& ft/'"
     "m/' feet|units -q|sed 's/^[[:space:]]*\* //"
     "/^[[:space:]]*\//d'"
 
